### PR TITLE
[Insights]Add custom reports doc

### PR DESCRIPTION
**Pull request by ruwiniwj**
## Purpose
Add custom reports doc

Fixes https://github.com/wso2-enterprise/choreo/issues/12927

Screenshot 2022-05-04 at 12 07 26
Screenshot 2022-05-04 at 12 07 33
Screenshot 2022-05-04 at 12 07 46

### CODE REVIEW

**Review by fazlan-nazeem — APPROVED** [review]

File: en/docs/insights/generate-custom-reports.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+2. Select metric(s) from the **Metrics** drop-down selector.
+
+    ![Generate report](../assets/img/insights/custom-reports-step-2-to-4.png){.cInlineImage-full}
+
+3. Select a minimum of 1 and a maximum of 3 group-by field(s) from the **Group By** drop down selector.
+
+4. Set the order of the group-by filters by drag and drop to determine the grouping order of the selected metric(s).
+
+5. After you generate the custom report, there will be a drop down selector per each selected group-by field in step 2 which lists the available values in the group-by field. You can change the group-by field(s) to change the grouping-criteria of the metric(s).
```
Comment (shaniR) [inline en/docs/insights/generate-custom-reports.md:19]: Once you determine the order of the group-by fields, you can set values for each group-by field from the respective dropdown.

File: en/docs/insights/generate-custom-reports.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Generate Custom Reports
+
+Choreo insights allows you to generate custom reports to view the information that is important to you to make timely decisions for the betterment of your business. With custom reports, you can generate reports for a set of metrics of your choice and aggregate results by specific fields. Once generated, the metrics can be filtered out using any selected group-by field(s). Custom reports supports three visualizations: overtime charts, pie charts, and tables.
```
Comment (shaniR) [inline en/docs/insights/generate-custom-reports.md:3]: Choreo insights allow you to generate custom reports to view the information that is important to you to make timely decisions for the betterment of your business. With custom reports, you can generate reports for a set of metrics of your choice and aggregate results by specific fields. Once generated, the metrics can be filtered out using any selected group-by filters. Custom reports support three visualizations: overtime charts, pie charts, and tables.

**Review by shaniR — CHANGES_REQUESTED** [review]

**Review by shaniR — APPROVED** [review]

**Comment by ruwiniwj** [comment]
Closing PR due to an issue with email associated with commit.

**Comment by ruwiniwj** [comment]
Reopened with https://github.com/wso2/docs-choreo-dev/pull/514

**Pull request closed without merging**